qa/automation/CareerPortal.Tests.Scenarios/1.Send your CV/Send CV.feature

```diff
@@ -0,0 +1,275 @@
+﻿Feature: Sending CV Form
```

Comment:
I enjoyed reviewing your BDD-scenario! You did a great Job! 
The main issues I have commented on:
1) Verbs of perception (see is repeated multiple times) should be replaced
2) Sometimes there are several When steps